services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImpl.java

```diff
@@ -165,6 +174,10 @@ public News updateNews(News news, String updater, Boolean post, boolean publish)
       newsTargetingService.deleteNewsTargets(news.getId(), updater);
       newsTargetingService.saveNewsTarget(news.getId(), StringUtils.equals(news.getPublicationState(), PublicationDefaultStates.STAGED), news.getTargets(), updater);
```

Comment:
IMO You need to create only one newsTarget metadata with a map of properties staged and achieved and not a metadata with each property. For that you need to modify metadata service layer in social too in order to manage a new query with a map of properties